### ISSUE TITLE
How to set the correct timezone for docker deployments

### ISSUE THREAD

**Issue by pixelchrome** [pgadmin-org/pgadmin4#7000]
**_Please note that security bugs or issues should be reported to security@pgadmin.org._**

**Describe the bug**

I want to set the timezone for a docker deployment. Working with the following in my `docker-compose.yml` doesn't help:

```
    volumes:
      - /etc/timezone:/etc/timezone:ro
```

or setting 

```
    environment:
...
      - TZ=Europe/Berlin
```

**To Reproduce**

see the above two settings I've used in `docker-compose.yml`

**Expected behavior**

It should be possible to change the timezone

**Error message**

**Screenshots**

Example of my postgres container

```sh
docker exec -it postgres-postgres-1 date
Fri Nov 24 14:03:20 CET 2023
```

same for the pgadmin container (after trying the described settings in `docker-compose.yml`)

```
docker exec -it pgadmin-postgres-1 date
Fri Nov 24 13:03:22 UTC 2023
```

**Desktop (please complete the following information):**

Latest (c06a45c0591a)

**Additional context**

As this container is based on Alpine Linux cat /etc/os-release (PRETTY_NAME="Alpine Linux v3.18") maybe you can add 'tzdata' to the packages. Then a simple mount ( /etc/timezone:/etc/timezone:ro ) can do the trick.

See https://wiki.alpinelinux.org/wiki/Setting_the_timezone

**Comment by pixelchrome** [comment]
It is now fixed with PR #7001

**Comment by pixelchrome** [comment]
Done
